### ISSUE TITLE
Boolean Toggle Variant

### ISSUE THREAD

**Issue by n1ru4l** [pmndrs/leva#199]
Sometimes it is nice to show a toggle instead of a checkbox. This could either be solved in core via a configuration option or as a plugin.

**Comment by dbismut** [comment]
I agree. Also I think we're using the label trick to style the checkbox but I think you can do it without it now.
